### ISSUE TITLE
Editor font size won't change? (Ubuntu)

### ISSUE THREAD

**Issue by pinkyblinky** [onivim/oni#2248]
Oni Version: 0.3.4
Neovim Version (Linux only): 0.3.4
Operating System: ubuntu 16.04 LTS

#### Describe your issue
Hello all! 
I want to increase the editor font size.
#### Expected behaviour
When I configured other options (ui.fontSize for instance), as soon as I saved the altered value the changes took effect. So when I repeated the process for editor.fontSize, I expected the same behaviour.
#### Actual behaviour
What actually happened was... nothing :(. I followed the advice on issue #2033 (regenerating the config.js and config.tsx) but the changes still never seemed to take effect. Please help :'(

![oniissue](https://user-images.githubusercontent.com/21164080/40565468-b8e30c64-603a-11e8-8c6e-3b1e6d067bf0.png)

#### Steps to reproduce
Try changing the editor.fontSize in config.tsx

**Comment by pinkyblinky** [comment]
P.S. I have Oni installed on both Mac and Windows as well. Neither seemed to suffer from this issue. Perhaps this is something specific to the linux distribution?

**Comment by badosu** [comment]
@pinkyblinky Seems like a parse error on line 19

**Comment by pinkyblinky** [comment]
Hmm... using the '%' key, it appears that this bracket is paired with the
one on line 5. I tried taking it out just to see but got a 'Unexpected
token export' from ~/.config/oni/config.js

**Comment by bryphe** [comment]
Thanks for investigating @pinkyblinky !

> Perhaps this is something specific to the linux distribution?

That does sound like the case, given that it doesn't reproduce on Windows / Mac.

We could try looking at the logs:
- `Control+Shift+P` -> `Debug: Open Developer Tools`
- In the console, run: `Oni.log.enableDebugLogging()`:
![image](https://user-images.githubusercontent.com/13532591/40571415-3427b124-604d-11e8-852d-661f98d126a6.png)
- Make a configuration change and save. You should see these two lines in the developer tools:
![image](https://user-images.githubusercontent.com/13532591/40571422-59353838-604d-11e8-8a19-5a6914ec573e.png)
Would be curious if one or both aren't showing up.

If you're up for debugging, this would be a good place to toss a breakpoint:
https://github.com/onivim/oni/blob/master/browser/src/Services/Configuration/ConfigurationEditor.ts#L76

This file is responsible for listening for configuration changes / transpiling / picking up the new config, so would be a good place to look.

**Comment by pinkyblinky** [comment]
Thank you for the prompt response :)

I opened the debug area as you suggested and then modified and saved the config.tsx file. Both of the entries you highlighted appear to be there:

![editconfiglog](https://user-images.githubusercontent.com/21164080/40580120-74a292aa-6106-11e8-8ad8-57e9bb8d1d4e.png)

I will look into debugging...

Interesting side note, I did a fresh install of ubuntu 16.04 LTS on a virtual box on windows and then installed Neovim and Oni. Then I tried to update the configuration there, and it actually behaved as expected, no bugs. Perhaps it is some odd setting I've tripped over in my current system...

I will look into building Oni from source and throwing in that breakpoint...

**Comment by pinkyblinky** [comment]
Ok, so I built Oni from source on my Ubuntu machine. And after starting it with 'yarn run launch' I altered the font size in the config.tsx file and it actually altered its appearance accordingly. Not sure exactly why the installed .deb version of Oni exhibited this bug specifically on my machine (but on others it did not), yet the built-from-source version worked perfectly fine on the very same machine. Anyway, I do now have a version of Oni that seems to be working. So I guess we can close this issue now?

Thank you everyone for your feedback and support on this issue :) I appreciate it!
